proposals/2024-04-10-native-support-for-info-metrics-metadata.md

```diff
@@ -0,0 +1,112 @@
+# Add 1st class feature to PromQL for handling info type metrics
```

Comment:
I don't think "target info" metrics would be fitting, since it's about info metrics in general, not just `target_info`. Also, the current title is geared towards the `info` PromQL function, rather than how to implement support for it (native info metric samples are a way to implement support).

Reply:
@beorn7 WDYT?

Reply:
Yeah, this is definitely not restricted to target info.

How about "Simplify joins with info metrics in PromQL"?

Reply:
Followed your suggestion @beorn7.